RadixWallet/Clients/ProfileStore/ProfileStore.swift

```diff
@@ -1,235 +1,99 @@
 import Sargon
 
 // MARK: - ProfileStore
-/// An in-memory holder of the app's `Profile` which syncs changes to *Keychain* and
-/// needed state to *UserDefaults* (activeProfileID). If user has iCloud Keychain sync
-/// enabled (we can't determine that) and has not disabled `Profile` cloud sync then
-/// iOS also syncs updates of `Profile` to iCloud via *Keychain*.
-///
-/// If a Profile successfully was loaded from Keychain it will be used an `Main` part
-/// of app is openened by `Splash`. If no existing Profile was found then a new one
-/// alongside a new (babylon) `.device` FactorSource, both persisted into *Keychain*,
-/// and user is pushed to `Onboarding`, to create a first account.
-///
-/// `ProfileStore` is an `actor` so that the in-memory `Profile` is protected against
-/// data races, however, it is not a "client" (TCA Dependency), rather it should be used by clients,
-/// and only by clients, not by Reducers directly, since it is quite low level.
-///
-/// This "public interface" (method meant to be used by the clients) is:
-///
-/// 	var profile: Profile { get }
-///		func values() -> AnyAsyncSequence<Profile>
-///	 	func finishedOnboarding() async
-///     func finishOnboarding(with _: AccountsRecoveredFromScanningUsingMnemonic) async throws
-///	 	func importProfile(_ s: Profile) throws
-///	 	func deleteProfile(keepInICloudIfPresent: Bool) throws
-/// 	func updating<T>(_ t: (inout Profile) async throws -> T) async throws -> T
-///     func claimOwnership(of profile: inout Profile)
-///
-/// The app is suppose to call `unlockedApp` after user has authenticated from `Splash`, which
-/// will emit any Profile ownership conflict if needed, and returns the newly claimed Profile that had
-/// ownership conflict if user chose that, else an entirely new Profile is user choses "Clear wallet on other Phone".
-///
-/// The app is suppose to call `finishedOnboarding` if user just finished onboarding a new wallet, it will
-/// async read `device.name` and `device.model` and update the Profile's header's `creatingDevice`
-/// and `lastUsedOnDevice` to use these values.
-///
-/// If a user creates a Profile using Account Recovery Scan, app is supposed to call `finishOnboarding:with`,
-/// and create a mainnet `network` **even if AccountsRecoveredFromScanningUsingMnemonic.accounts** is empty,
-/// ensuring next time the user starts app, they will not be met with onboarding..., but rather `Home`, with
-/// an empty account list.
-///
-/// And then a lot of sugar/convenience AsyncSequences using `values` but mapping to other
-/// values inside of `Profile`, e.g.:
-///
-/// 	func accountValues() async -> AnyAsyncSequence<Accounts>
-///
-/// And similar async sequences.
-///
 public final actor ProfileStore {
```

Comment:
maybe a word about future directions? In the future this will be deleted right?

Reply:
Added.

> In the future this will be deleted right?

Likely, we would potentially call Sargon directly in the respective clients, instead of going through the ProfileStore.